make-util.js

```diff
@@ -444,7 +444,11 @@ var downloadArchive = function (url, omitExtensionCheck) {
 
     // skip if already downloaded and extracted
     var scrubbedUrl = url.replace(/[/\:?]/g, '_');
-    var targetPath = path.join(downloadPath, 'archive', scrubbedUrl);
+
+    var crypto = require('crypto');
```

Comment:
Add the owners of this file (if different) to the PR as well. I think it looks good.

Reply:
hi @tintse-thxsky-MSFT, could you please describe the case you fixed?